deltachat-ffi/src/lib.rs

```diff
@@ -3972,6 +3972,20 @@ pub unsafe extern "C" fn dc_contact_is_verified(contact: *mut dc_contact_t) -> l
         .unwrap_or_default() as libc::c_int
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn dc_contact_verifier(contact: *mut dc_contact_t) -> *mut libc::c_char {
```

Comment:
```suggestion
pub unsafe extern "C" fn dc_contact_get_verifier(contact: *mut dc_contact_t) -> *mut libc::c_char {
```

we should stay with conventions already in use.